src/youtube/players/YoutubeMobileArticleVideoPlayer.tsx

```diff
@@ -34,7 +32,7 @@ const MobileArticleVideoWrapper: React.FC<MobileArticleVideoWrapperProps> = ({
 						: styles.mobileArticleVideoWrapperIsNotScrollPlayer,
 				)}
 				style={{
-					...(isScrollPlayer && { top: topPosition }),
+					...(isScrollPlayer && { top: scrollTopPosition }),
```

Comment:
Considering the type, it seems that `scrollTopPosition` could potentially become `undefined` in both this context and in `src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx`. Do you think we should address this situation?

Reply:
The value in the context should always be set but even if it is undefined it will just not have `top` css property. I think it's fine